### PR TITLE
Redesign navigation

### PR DESCRIPTION
**Pull request by manuthecoder**
# Summary  
This PR ushers HCB into *UI3*, introducing a fully redesigned navigation experience for desktop, mobile, and tablet platforms.  

#### General Changes  
- **Account Information**:  
  - Moved to the bottom of the screen on desktop.  
  - Positioned in the top-right corner on mobile.  
- **Sidebar Enhancements**:  
  - Subheaders removed for a cleaner look.  
  - **Settings Navigation**:  
    - Now accessible via a dropdown menu in the sidebar.  
    - Page title dynamically updates to reflect the current settings page.  
    - "Settings" header displayed in a subtle eyebrow format.  
- **Breakpoints**: Layout reconfigured for more adaptive responsiveness.  
- **Visual Tweaks**:  
  - Added `/` separator between the HCB logo and organization switcher.  
  - Sidebar and top app bar now have subtle blur effect

#### Platform-Specific Improvements:  

- **Desktop**:  
  - Sidebar is now fixed for better usability.  

- **Mobile**:  
  - Introduced Dylan’s pill navigation for a streamlined user flow.  

- **Tablet**:  
  - Adopted a wider version of the mobile layout for better readability. 

# Screenshots
| User home | Event home | Event settings |
|--------|--------|--------|
| image| image| image | 

|  | | | | |
|--------|--------|--------|--------|--------|
| image | image | image | image | image | 

image

### CODE REVIEW

File: app/views/events/_nav.html.erb
Comment (sampoder) [inline app/views/events/_nav.html.erb:1]: Is it possible to keep the tooltips?

**Comment by sampoder** [comment]
Screenshot 2024-10-31 at 2 35 16 PM

Would it be possible to put this content:

Screenshot 2024-10-31 at 2 36 01 PM

In the top right on mobile?

**Review by garyhtou — CHANGES_REQUESTED** [review]
Some feedback:
- The contents of the Settings dropdown should not wrap
  image
- Settings link on Mobile doesn't work
- Remove the subheaders within the navigation. See this mock up for reference (ignore the icons in screenshot since they're out of date)
  image

----

For later:
- Add slash next to the HCB logo
  image
- Horizontal dividing link above user section doesn't touch the vertical line
  image
  - The change log button should be closer to the support button (gap is too big right now)
- The headway widget covers the button that opens it. The widget should open higher.
  image
- There is a little bit of horizontal scroll on mobile
  image
- On mobile when scrolling up, the nav moves a bit. It should be fixed to the view port.
  image

**Comment by leowilkin** [comment]
Can we retain the section headings on the sidebar for desktop? On onboardings and in using the platform I think its much more intuitive if it's broken down into categories like Spending, receiving etc.

**Comment by garyhtou** [comment]
> Can we retain the section headings on the sidebar for desktop? On onboardings and in using the platform I think its much more intuitive if it's broken down into categories like Spending, receiving etc.

This is something you can pitch to @dwahbe! Feel free to start a thread in `#hcb-team` if other operations folks have thoughts on this too.

**Comment by garyhtou** [comment]
Some nav items were added to the settings sub nav, but wasn't removed to the main nav

image

For example, Team should be removed from the main nav

**Pull request merged**